src/main/java/com/example/EnumDemo/model/DressEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.example.EnumDemo.model;
+
+import com.example.EnumDemo.enumPackage.Color;
+import com.example.EnumDemo.enumPackage.DressType;
+import com.example.EnumDemo.enumPackage.Size;
+import jakarta.persistence.*;
+import lombok.*;
+@Entity
+@Table(name = "DressesTable")
```

Comment:
There is no need to add Table suffix in the table name. 
Also, table and column names should be 
singular
use snake case

Reply:
Done